app/controllers/application_controller.rb

```diff
@@ -10,4 +10,10 @@ def authentication_callback
   def authenticate!
     redirect_to '/auth/nyulibraries' unless session[:auth] && (session[:auth]['credentials']['expires'] == false || session[:auth]['credentials']['expires_at'] > Time.now.to_i)
```

Comment:
Yeah, I had definitely considered that, but the massive `unless` was temporary until we decided what would be considered "authenticated" to authz. In the next PR I intend to move that to a new function.